DorisProject/src/app/app.module.ts

```diff
@@ -18,6 +18,16 @@ import { ModalComponent } from './pages/modal/modal.component';
 import { MaterialModule } from './material-module';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { NumberOnlyDirective } from './pages/directives/number-only.directive';
+import {MatIconModule} from '@angular/material/icon';
```

Comment:
Codacy has a fix for the issue: [Replace `MatIconModule` with `·MatIconModule·`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9275970)
```suggestion
import { MatIconModule } from '@angular/material/icon';
```